tests/ui/c-variadic/same-program-multiple-abis-arm.rs

```diff
@@ -0,0 +1,62 @@
+#![feature(extended_varargs_abi_support)]
+//@ run-pass
+//@ only-arm-unknown-linux-gnueabihf
```

Comment:
Why only that one specific target? (And not all arm targets, or so)

Reply:
>  we're using floats because that is one case where we know aapcs and C are different

Please always put such choices into comments! The chance that someone else reading the code will realize this without a comment is probably negligible.